bot/code_review_bot/__init__.py

```diff
@@ -32,17 +32,42 @@ def __init__(self, code, message):
 class Issue(abc.ABC):
     """
     Common reported issue interface
-
-    Several properties are also needed:
-    - path: Source file path relative to repo
-    - line: Line where the issue begins
-    - nb_lines: Number of lines affected by the issue
     """
 
-    lines_hash = None
     is_new = False
     revision = None
 
+    def __init__(
+        self,
+        analyzer: str,
+        revision,
+        path: str,
+        line: int,
+        nb_lines: int,
+        check: str,
```

Comment:
I'd find a better name for this, maybe "rule"?

Reply:
It was rule in one analyzer, and check everywhere else

Reply:
As discussed, let's keep `check` for now